src/main/java/lotterymachine/model/WinningLottery.java

```diff
@@ -0,0 +1,51 @@
+package lotterymachine.model;
+
+import java.util.Arrays;
+import java.util.EnumMap;
+import java.util.Map;
+
+public enum WinningLottery {
+    THREE(3, 5000),
+    FOUR(4, 50000),
+    FIVE(5, 150000),
```

Comment:
```suggestion
    THREE(3, 5_000),
    FOUR(4, 50_000),
    FIVE(5, 150_000),
```
언더스코어를 넣으면 가독성을 챙길 수 있을 것 같아요 🙂
https://docs.oracle.com/javase/7/docs/technotes/guides/language/underscores-literals.html

Reply:
이런 방법이 있었군요...🤭🤭
감사합니다!